penguin/integrations/mcp/server_tools/ituv.py

```
@@ -0,0 +1,618 @@
+        return {
+            "status": "applied",
+            "task_id": task_id,
+            "action": "mark_ready_for_review",
+            "dry_run": False,
+            "before": before,
+            "after": serialize_task_payload(updated),
+        }
+
+    def record_artifact(arguments: dict[str, Any]) -> dict[str, Any]:
+        task_id = _required_str(arguments, "task_id")
+        dry_run = _optional_bool(arguments.get("dry_run"), True)
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)